### PR TITLE
Machine translate SSO templates

### PR DESCRIPTION
**Pull request by lukecamelo**
## Summary

Adds `message_[locale code].properties` files for all the languages supported by MyCharlie, populated with machine translations courtesy of AWS Translate.

Also enables the 7 languages in question in each of the MBTA themes:
- English [`en`]
- Español (Spanish) [`es`]
- Kreyòl Ayisyen (Haitian Creole) [`ht`]
- Português (Portuguese) [`pt`]
- Tiếng Việt (Vietnamese) [`vi`]
- 中文简体 (Chinese Simplified) [`zh-CN`]
- 中文繁體 (Chinese Traditional) [`zh-TW`]

## Notes

This is an effort I started almost exactly a month ago in May before shelving it to direct my focus to other work. Between then and now it seems there has been some additional work done in the localization department (#30), at least for `files/templates/mbta/account`,  where I see a whole bunch of new localized `.properties` files.  A couple of the supported languages were missing, so I added those. 

However, it seems that the format of the rest of the message files in that directory as added by @jurakp is slightly different, so my additions will probably need updating in some way. The same might be true of the `.properties` files I have created in the `files/templates/mbta/email` and `files/templates/mbta/login` directories, so I'm opening this PR as a draft first.

Update (7/10/2024): I (@nihonjinrxs) have tested these machine translation files in a local copy of Keycloak configured similarly to Keycloak Dev, and they work as expected. There may be some missing strings for error messages and such, but those can be added in as they are reported. (They are difficult to find directly, as they don't show up as `msg(...)` calls in the template, and only appear if the relevant error happens.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207564945811063

### CODE REVIEW

**Review by arkadyan — COMMENTED** [review]
question: There are no changes to the `messages/messages_es.properties` file. Is that ok?

Otherwise, this seems fine from the MyCharlie point of view. It would be good to still get a look from Kris, who knows more about the Keycloak side.

File: files/templates/mbta/account/messages/messages_en.properties [hunk 1]
```diff
@@ -1,3 +1,11 @@
+locale_en=English
+locale_es=Español (Spanish)
+locale_ht=Kreyòl Ayisyen (Haitian Creole)
+locale_pt=Português (Portuguese)
+locale_vi=Tiếng Việt (Vietnamese)
+locale_zh-CN=中文简体 (Chinese Simplified)
+locale_zh-TW=中文繁體 (Chinese Traditional)
```
Comment (Adzz) [inline files/templates/mbta/account/messages/messages_en.properties:10]: these are underscores but elsewhere we use a dash, which is correct? Or are they different in different places?

**Comment by Adzz** [comment]
This is fine for now as it's pretty static, but it looks like it wouldn't be hard to write a script that does similar to what we do for the PO files here. We should consider that in the future if maintenance here becomes hard.

File: files/templates/mbta/account/messages/messages_en.properties (hunk 1 above)
Comment (nihonjinrxs) [reply files/templates/mbta/account/messages/messages_en.properties:10]: These are what Keycloak expects, and I've tested locally that they work as expected.

Comment (nihonjinrxs) [reply files/templates/mbta/account/messages/messages_en.properties:10]: For the language codes, the standard is `-` (hyphen) between segments of the language code. Keycloak expects `locale_<language code>` as the key here.

**Comment by nihonjinrxs** [comment]
> question: There are no changes to the `messages/messages_es.properties` file. Is that ok?

I think you're referring to the `account` theme's Spanish properties file. Yes, that theme had a pretty complete Spanish translation already (and many others) added prior, and it also is not a user-facing theme for us. (Seems mostly to be an artifact of prior ideas about how to use Keycloak at MBTA.)

If you're in #mbta-integsoft on Slack, we've got [a thread](https://mbta.slack.com/archives/C02GYEJTDKJ/p1720712279654719) asking about the origin of those `account` theme translations, as they got added in a PR (#30) by Integsoft then.

**Comment by arkadyan** [comment]
> I think you're referring to the `account` theme's Spanish properties file.

Oops, yep, meant to include that bit in the path. Thanks, just wanted to confirm.

**Review by arkadyan — DISMISSED** [review]

File: files/templates/mbta/account/messages/messages_en.properties (hunk 1 above)
Comment (Adzz) [reply files/templates/mbta/account/messages/messages_en.properties:10]: ah yes I misread

**Review by Adzz — DISMISSED** [review]
Obvs I can't test that the translations are correct but looks good

File: files/templates/mbta/account/messages/messages_en.properties (hunk 1 above)
Comment (krisrjohnson21) [reply files/templates/mbta/account/messages/messages_en.properties:10]: Question (non-blocking): Is it worth adding a code comment to the effect of `Keycloak expects locale_<language code> as the key`?

**Review by krisrjohnson21 — COMMENTED** [review]
This LGTM with one question. I don't see a `files/templates/mbta/account/messages/messages_es.properties` file (Spanish). Should there be one?

**Comment by krisrjohnson21** [comment]
> This LGTM with one question. I don't see a `files/templates/mbta/account/messages/messages_es.properties` file (Spanish). Should there be one?

Nevermind, I see Matt's question above that was already answered. Approving!

**Review by krisrjohnson21 — DISMISSED** [review]

**Comment by nihonjinrxs** [comment]
Added a comment regarding the language codes to the `en` properties files in [fc24d76](https://github.com/mbta/keycloak-deploy/pull/33/commits/fc24d76ad76e5fea79fd13408f36cf476ea46cc8)

**Review by krisrjohnson21 — APPROVED** [review]
🧼

**Pull request merged**